protocol/src/main/resources/message.schema.json

```diff
@@ -5,16 +5,16 @@
       "type": "object",
       "properties": {
         "from": {
-          "$ref": "https://tbdex.io/definitions.json#/definitions/did",
```

Comment:
including the `https://tbdex.io/` led to `FileNotFoundException`

Reply:
see tbdex change here: https://github.com/TBD54566975/tbdex/pull/166